src/App.js

```diff
@@ -42,9 +87,34 @@ class App extends React.Component {
           {this.state.guessedLetters.length > 0
```

Comment:
```suggestion
          {this.state.guessedLetters.length
```

0 is falsy. Otherwise numbers are truthy